v1/openings/models/opening.py

```diff
@@ -4,20 +4,24 @@
 from django.db import models
 from thenewboston.models.created_modified import CreatedModified
 
+from v1.contributors.models import Contributor
+from v1.teams.models import Team
+from .responsibility import Responsibility
+from .skill import Skill
+
 
 class Opening(CreatedModified):
     uuid = models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True)
-    title = models.CharField(max_length=250)
+    reports_to = models.ManyToManyField(Contributor)
```

Comment:
I'd suggest to always use string imports here
Pycharm supports that perfectly
and using string imports in the model's foreign key definition will prevent circular deps problem